### PR TITLE
restyle ls to work like exa

### PR DESCRIPTION
**Pull request by maximsmol**
Restyled `ls` to this:
Screen Shot 2022-07-15 at 11 14 44 PM

Inspired by [`exa`'s](https://the.exa.website/) color scheme:
Screen Shot 2022-07-15 at 11 16 46 PM

### CODE REVIEW

File: latch_cli/main.py [hunk 1]
```diff
@@ -138,57 +139,87 @@ def cp(source_file: str, destination_file: str):
...
+            click.echo(f"{remote_directory}:")
+
         try:
             output = ls(remote_directory)
         except Exception as e:
             click.secho(
                 f"Unable to display contents of {remote_directory}: {str(e)}", fg="red"
             )
             continue
+        output.sort(key=lambda row: row["name"])
```
Comment (ayushkamat) [inline latch_cli/main.py:162]: can you sort so that all directories appear together

File: latch_cli/utils.py [hunk 2]
```diff
@@ -74,9 +74,23 @@ def _normalize_remote_path(remote_path: str):
     return remote_path
 
 
-def with_si_suffix(num, suffix="B"):
-    for unit in ["", "K", "M", "G", "T", "P", "E", "Z"]:
+def _si_number_strings(num):
+    for unit in ["", "k", "M", "G", "T", "P", "E", "Z"]:
```
Comment (ayushkamat) [inline latch_cli/utils.py:78]: why lowercase k

**Review by ayushkamat — CHANGES_REQUESTED** [review]
lgtm for the most part, only real thing is that i would like it to be first sorted with directories appearing together

File: latch_cli/utils.py (hunk 2 above)
Comment (maximsmol) [reply latch_cli/utils.py:78]: SI prefix for "kilo" is lowercase "k" :shrug:

**Review by ayushkamat — APPROVED** [review]
sicco mode

**Pull request merged**